languages/go/internal/cinterface/bitwarden_library.go

```diff
@@ -0,0 +1,58 @@
+//go:build !custom
+// +build !custom
+
```

Comment:
With the custom file gone, can we remove these comments?

Reply:
Removed